src/lib/integration_test_cloud_engine/kubernetes_network.ml

```diff
@@ -131,27 +131,12 @@ module Node = struct
           sendTestSnapp(parties: $parties) {
                snapp { id
                        hash
-                       nonce
                        failureReason
                      }
              }
          }
        |}]
 
-    module Send_snapp =
```

Comment:
This is actually a real pain to use because the graphql_ppx requires all the arguments to be passed as objects.
I think to be able to use something like this either:
(a) we'd want to introduce a `to_graphql_input_obj`, but since it's an object we'd have to generate it in a ppx, or
better, (b) we fork the graphql_ppx to provide a way to hand JSON directly instead of this object to the query (which internally gets serialized to JSON immediately anyway)

However, I'd argue either of these is a waste of time when we could just send the JSON directly through the Send_test_snapp escape hatch anyway

Reply:
The real endpoint will be properly tested in the full mina-signer-snarkyjs loop so we don't need to worry about leaving that untested